spinta/datasets/keymaps/sync.py

```diff
@@ -81,14 +82,15 @@ def sync_keymap(
                 counters[model.model_type()] = model_counters
 
             for row in data:
-                if row['_op'] == 'insert':
+                if row['_op'] == 'insert' and not dry_run:
```

Comment:
Neverta būtų iškelti`dry_run` patikrinimą į pradžią, tai yra, jei `counters` nebus skaičiuojami (kaip dabar padarei), tada tiesiog pakeisti if check į `if not dry_run and status_code == 200`, nebent tikslas, kad prasisuktu counters, tada reiktų keisti šiek tiek logiką ?